samples/macOS/clean/ls.mdiff

```diff
@@ -1,16 +1,18 @@
-## Changed: . [⚠️ MEDIUM → ✅ LOW]
```

Comment:
This should be a file change - but instead of `.`, it should say the name of the new filename.

Reply:
Addressed in [`dfc5173` (#356)](https://github.com/chainguard-dev/bincapz/pull/356/commits/dfc517390378280a98a62403b6d9c29f3d6075e3).